tests/end-to-end/backup-restore-test/README.md

```diff
@@ -0,0 +1,81 @@
+# End-to-end Backup and Restore Tests
+
+## Overview
+
+This project contains end-to-end upgrade tests for the Kyma installation. The tests run daily on Prow to validate if the backup and restore process works for all components.
```

Comment:
`This project contains end-to-end backup tests...` 😄

Reply:
woops :P changed!